tsl/test/expected/remote_txn.out

```diff
@@ -426,10 +426,9 @@ NOTICE:  [loopback]:  INSERT INTO "S 1"."T 1" VALUES (10002,1,'bleh', '2001-01-0
  
 (1 row)
 
-COMMIT;
-WARNING:  kill event: pre-prepare-transaction
-WARNING:  transaction rollback on data node "loopback" failed
-ERROR:  [loopback]: SSL connection has been closed unexpectedly
+-- since the error message might differ between environments we suppress normal output and print sqlstate
+\echo :SQLSTATE
+08006
```

Comment:
It would be good to know what the SQLSTATE means in this case. Now it is very opaque. Would it possible to use, e.g., `LAST_ERROR_SQLSTATE` or `LAST_ERROR_MESSAGE` instead?

Reply:
LAST_ERROR_SQLSTATE does not get reset on successful queries so it would be wrong to use because we want to get the sqlstate of the commit and not any command that happened to trigger last error. LAST_ERROR_MESSAGE is platform dependent and is basically what this PR is trying to avoid using.
I'll add comment which state is expected. You can look up the states here: https://www.postgresql.org/docs/current/errcodes-appendix.html